treeopt/treeOpt.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import os
+
+#Import of treeopt submodules
+import modules.sampling as sampling
```

Comment:
I would move the modules one directory level up, directly under treeopt. Then you can use `import treeopt.sampling as sampling`, which is a bit more pythonic.